packages/edit-post/src/store/selectors.js

```diff
@@ -367,6 +367,17 @@ export function isEditingTemplate( state ) {
 	return state.isEditingTemplate;
 }
 
+/**
+ * Returns true if meta boxes are initialized.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether meta boxes are initialized.
+ */
+export function metaBoxesInitialized( state ) {
```

Comment:
Can we maybe rename this one  `areMetaBoxesInitialized`, `hasInitializedMetaboxes` or something like that to match the naming scheme of the other selectors?

Reply:
Done